lib/ansible/module_utils/network/vyos/vyos.py

```diff
@@ -35,7 +35,7 @@
 
 vyos_provider_spec = {
     'host': dict(),
-    'port': dict(type='int'),
+    'port': dict(type='int', default=22),
```

Comment:
Do we still need this? We're skipping the old modules in validate-modules test.

Reply:
No we don't need this as we are skipping the old modules in validation test .This has been removed now.